src/forms/Checkbox.jsx

```diff
@@ -1,33 +1,24 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import cx from 'classnames';
-import Flex from '../layout/Flex';
-import FlexItem from '../layout/FlexItem';
-import Icon from '../media/Icon';
-
-export const FAUX_TOGGLE_CLASS = 'fauxToggle';
-export const FOCUSED_CHECKBOX_CLASS = 'focused';
-export const DISABLED_CHECKBOX_CLASS = 'disabled';
+import { default as SwarmCheckbox } from '@meetup/swarm-components/lib/Checkbox';
 
 /**
  * @module Checkbox
  */
 class Checkbox extends React.PureComponent {
 	constructor(props) {
 		super(props);
-		const state = {
-			focused: false,
+
+		this.state = {
```

Comment:
can we use class properties in MWC? If so, we can avoid the constructor

```js
class Checkbox extends React.PureComponent {
  state = this.props.checked || false;
  onChange = e => ...; // moved up from line 36
  ///...
```

Reply:
They should be supported in mwc

Reply:
I guess the question is, does it matter? :) They're already classes and our goal is eventually to deprecate meetup-web-components entirely. What would this gain us?